tests/QueryBuilder.test.ts

```diff
@@ -2942,7 +2942,7 @@ describe('QueryBuilder', () => {
     spy.mockRestore();
   });
 
-  test('limit of 0 limits results to 0',()=>{
+  test('limit of 0 limits results to 0', ()=>{
```

Comment:
hmm this is missing some spaces as well around the arrow, but that must have some eslint rule as well... damn, I though I got it set up well, but apparently I am missing some very basic rules in there.

```suggestion
  test('limit of 0 limits results to 0', () => {
```